### DISCUSSION TITLE
Comment out Templater syntax in Obsidian

### DISCUSSION THREAD

**Discussion by sccardais** [SilentVoid13/Templater#1333] [Help]
I'd like to create a page in Obsidian with examples of Templater syntaxes.

Each example would use the same syntax but one would show the result. The other would show the syntax. e.g To get this Result, use this Syntax.

When I use the Replace command, it replaces both. How can I comment out the one that I don't want Replaced? 

I'm sure theres a way (and probably a simple way!) but I've tried quotations and // but these don't prevent replacement.

Thanks for any help.

**Comment by Zachatoo** [comment]
You can escape the commands with the escape character `\` like so
```
<\% ... %>

**Comment by sccardais** [comment]
Thanks very much for the fast response.

ScottC
Saluda, NC
